Documentation/Signoffs/dylanrobbins-mold-signoff.md

```diff
@@ -0,0 +1,65 @@
+# Mold Module Signoff
+
+## Subsystem Function
+The function of the mold module subsystem is to detect if mold is likely to grow within the surrounding environment of the sensor. The SHT30 Temperature and Humidity Sensor will gather temperature and humdity levels and send its respective data to an ESP32-H2 micro-controller. The mold module will send data to the ESP32-H2 every hour to allow for computations of the peak and average values.
+
+![image](https://github.com/jacksonrwoodard/HouseHealthMonitoring/assets/104484972/3401a3b6-74a1-49af-a090-dfe94abc742c)
+
+
+## Constraints
+| No. | Constraints | Origin |
+| --- | ----------- | ------ |
+|  1  | Shall be able to detect temperature ranges between 55&deg; F - 85&deg; F and humidity levels exceeding 50% RH. | Project Team |
+|  2  | Shall be able to communicate through I2C protocol and send data every hour to its local ESP32-H2 transmitter. | Project Team |
+|  3  | Shall be protected and not exposed to harsh environmental conditions to prevent any damage to the sensor. | All External Stakeholders, Ethics, & Team Supervisor |
+|  4  | Shall give precise readings to properly evaluate and determine if mold like conditions are present. | Project Team |
```

Comment:
this is not specific enough.

Reply:
I have addressed that this is not sufficiently precise.

Reply:
I added specific reading ranges to be more precise in the constraint.